README.md

```diff
@@ -14,9 +14,8 @@
 
 ## Running your tests
 
-- To run a parallel tests, run `mvn test -P sample`
-- To run local tests, run `mvn test -P local`
-- To run a full suite of tests, run `mvn test -P suite`
```

Comment:
Let's keep suite as of now, people will come to know how to use multiple test files.